libvast/test/hash.cpp

```diff
@@ -32,7 +35,17 @@ auto inspect(Inspector& f, foo& x) {
   return f(x.a, x.b);
 }
 
-} // namespace <anonymous>
+template <class T, size_t N>
+std::string hexify(const std::array<T, N>& xs) {
+  std::stringstream ss;
```

Comment:
A stream seems a bit heavyweight for this task and the code. Did you consider using `byte_to_hex` instead?

```c++
  std::string result;
  for (auto x : xs) {
    auto [first, second] = byte_to_hex<policy::lowercase>(x);
    result += first;
    result += second;
  }
  return result;
```

I think a version like this is also easier to follow.

Reply:
I've factored `hexify` into a utility function. Now it's easier to reference.